src/components/Article.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Article({
+  title,
+  date,
+  text,
+}) {
+  return (
+    <div>
+      <h1>
+        {title}
+      </h1>
+      <span>
+        {date}
+      </span>
+      <p>
+        {text}
+      </p>
+    </div>
+  );
+}
```

Comment:
```suggestion
function Article({ title, date, text }) {
  return (
   <>
      <h1> {title} </h1>
      <span> {date} </span>
      <p> {text} </p>
    </>
  );
}
```
Так смотрится гораздо легче, не стоит всё переносить на новую строку. Так же лучше обернуть в не в div, а в пустой тег